modules/portworx_with_cloud_drive/script_sc.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
is the script required?

Reply:
@hkantare User can use this script to get the name of the storageclass